bin/inspec

```diff
@@ -106,5 +122,42 @@ class InspecCLI < Thor
   def version
     puts Inspec::VERSION
   end
+
+  private
+
+  def diagnose
+    return unless opts['diagnose']
+    puts "InSpec version: #{Inspec::VERSION}"
+    puts "Train version: #{Train::VERSION}"
+    puts 'Command line configuration:'
+    pp options
+    puts 'JSON configuration file:'
+    pp options_json
+    puts 'Merged configuration:'
+    pp opts
+  end
```

Comment:
What do you think about an additional newline?

Here's a use-case where this might be clearer:

```
./bin/inspec detect --diagnose
InSpec version: 0.9.5
Train version: 0.9.1
Command line configuration:
{"diagnose"=>true}
JSON configuration file:
{}
Merged configuration:
{"diagnose"=>true}
{"name":null,"family":"arch","release":"4.2.5-1-ARCH","arch":null}
```

The last line feels like a part of diagnose.